core/src/main/java/com/aem/demo/core/servlets/ServletForRssFeed.java

```diff
@@ -0,0 +1,58 @@
+/*
+ *  Copyright 2015 Adobe Systems Incorporated
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package com.aem.demo.core.servlets;
+
+import com.aem.demo.core.services.RssFeedService;
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.SlingHttpServletResponse;
+import org.apache.sling.api.servlets.HttpConstants;
+import org.apache.sling.api.servlets.SlingAllMethodsServlet;
+import org.apache.sling.api.servlets.SlingSafeMethodsServlet;
+import org.osgi.framework.Constants;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+import javax.servlet.Servlet;
+import java.io.IOException;
+
+/**
+ * Servlet that writes some sample content into the response. It is mounted for
+ * all resources of a specific Sling resource type. The
+ * {@link SlingSafeMethodsServlet} shall be used for HTTP methods that are
+ * idempotent. For write operations use the {@link SlingAllMethodsServlet}.
+ */
+@Component(service = Servlet.class,
+        property = {
+                Constants.SERVICE_DESCRIPTION + "=Simple Demo Servlet",
+                "sling.servlet.methods=" + HttpConstants.METHOD_GET,
+                "sling.servlet.paths=" + "/apps/aemtraining/servlet/updater"
+        })
+public class ServletForRssFeed extends SlingSafeMethodsServlet {
+
+    @Reference
+    private transient RssFeedService rssFeedService;
+
+
+
+    @Override
+    protected void doGet(final SlingHttpServletRequest request,
+                          final SlingHttpServletResponse response) throws IOException {
+        response.setContentType("application/json");
```

Comment:
Please be carefult with mime types. What you are currently outputting, is `text/html`. You will possibly output json when the service is completed. But anyway that is not necessary in the current case. Whoever / whatever reads the response will be satisfied with the response status 200, even without a text. 

"200" means all went well. And if there's an error, you change the status code to the code of an appropriate error

But for sure you can add some text to response if there's something particular to say. E.g., the concrete error message